room/compiler-processing/src/main/java/androidx/room/compiler/processing/XProcessingEnv.kt

```diff
@@ -33,6 +33,12 @@ import kotlin.reflect.KClass
 interface XProcessingEnv {
 
     val backend: Backend
+
+    /**
+     * The target language the processor is set to expect
+     */
```

Comment:
we don't have anything here yet but i think this is probably the best place to document behavior changes between the two. As we add them, we should start documenting them here (open for better suggestions though )